### PR TITLE
CTCP-1637: Add Replacement Identification Number Page

### PR DESCRIPTION
**Pull request by Reece-Carruthers**
Have an outstanding question regarding what happens when unknown option is selected, will discuss at stand-up. At the moment just have some placeholder text until we figure out how we want to go about it.

### CODE REVIEW

File: app/navigation/annotations/Incident.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.PARAMETER, ElementType.METHOD, ElementType.FIELD})
+@BindingAnnotation
+public @interface Incident {
+}
```
Comment (kevincoatsworth) [inline app/navigation/annotations/Incident.java:30]: You can delete this, the scaffold still generates an annotation file

File: conf/messages.en [hunk 2]
```diff
@@ -498,15 +498,33 @@ incident.equipment.seal.remove.error.required = Select yes if you want to remove
...
 incident.transportMeans.identification.trainNumber = Train number
+incident.transportMeans.identification.trainNumber.arg = train number
 incident.transportMeans.identification.regNumberRoadVehicle = Registration number of a road vehicle
+incident.transportMeans.identification.regNumberRoadVehicle.arg = registration number of a road vehicle
 incident.transportMeans.identification.regNumberRoadTrailer = Registration number of a road trailer
+incident.transportMeans.identification.regNumberRoadTrailer.arg = registration number of a road trailer
 incident.transportMeans.identification.regNumberAircraft = Registration number of an aircraft
+incident.transportMeans.identification.regNumberAircraft.arg = registration number of an aircraft
 incident.transportMeans.identification.europeanVesselIdNumber = European vessel identification number (ENI code)
+incident.transportMeans.identification.europeanVesselIdNumber.arg = European vessel identification number (ENI code)
```
Comment (kevincoatsworth) [inline conf/messages.en:519]: Should this one start lower case?

File: conf/prod.routes [hunk 3]
```diff
@@ -3,4 +3,6 @@
 
 ->         /                                                   health.Routes
 
+->         /manage-transit-movements/arrival                   app.incident.transportMeans.Routes
```
Comment (kevincoatsworth) [inline conf/prod.routes:6]: You can remove this line, we've got it in the app.incident.routes file

File: conf/messages.en (hunk 2 above)
Comment (Reece-Carruthers) [reply conf/messages.en:519]: That should be uppercase

**Review by kevincoatsworth — APPROVED** [review]

**Pull request merged**